database/migrations/2022_02_02_135907_create_consultations_participants_table.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateConsultationsParticipantsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('consultations_participants', function (Blueprint $table) {
+            $table->id();
+            $table->bigInteger('consultation_id')->index();
+            $table->bigInteger('user_id')->index();
+            $table->string('status');
```

Comment:
what is `status` ?

Reply:
status determines whether the participant is connected or disconnected

Reply:
Maybe add a default value to it `->default(Enum::VALUE)`

Reply:
Added enum to participant status. Default status is disconnected